snapcraft/plugins/go.py

```diff
@@ -39,6 +39,10 @@
       This entry tells the checked out `source` to live within a certain path
       within `GOPATH`.
       This is not needed and does not affect `go-packages`.
+
+    - go-buildtags:
+      (string)
```

Comment:
El 17/08/16 a las 11:55, Kyle Fazzari escribió:
> In snapcraft/plugins/go.py
> <https://github.com/snapcore/snapcraft/pull/733#discussion_r75139056>:
> 
>> @@ -39,6 +39,10 @@
>>        This entry tells the checked out `source` to live within a certain path
>>        within `GOPATH`.
>>        This is not needed and does not affect `go-packages`.
>> +
>> +    - go-buildtags:
>> +      (string)
> 
> Okay. In that case, should we make sure we stay true to the command-line
> options? i.e. call this |go-tags|?

go-tags feels confusing to me, but if both of you prefer it fine by me ;-)